src/admin-views/settings/tickets-commerce/stripe/connect/help-links/configuring.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * The Template for displaying the Tickets Commerce Stripe help links (configuring).
+ *
+ * @version 5.2.0
+ *
+ * @since   5.2.0
+ *
+ * @var Tribe__Tickets__Admin__Views                  $this                  [Global] Template object.
+ * @var string                                        $plugin_url            [Global] The plugin URL.
+ * @var TEC\Tickets\Commerce\Gateways\PayPal\Merchant $merchant              [Global] The merchant class.
+ * @var TEC\Tickets\Commerce\Gateways\PayPal\Signup   $signup                [Global] The Signup class.
```

Comment:
```suggestion
 * @version TBD
 *
 * @since   TBD
 *
 * @var Tribe__Tickets__Admin__Views                  $this                  [Global] Template object.
 * @var string                                        $plugin_url            [Global] The plugin URL.
 * @var TEC\Tickets\Commerce\Gateways\Stripe\Merchant $merchant              [Global] The merchant class.
 * @var TEC\Tickets\Commerce\Gateways\Stripe\Signup   $signup                [Global] The Signup class.
```